docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -233,9 +233,7 @@ question: |
 # It is easy to copy and edit this event without removing the version
 # information. We may consider using the form description, adding a 
 # thumbnail of the original form, etc. at some point.
-subquestion: |
-  [CENTER] [FILE seal.jpg, 30%, Logo of the Suffolk LIT Lab]
-  
+subquestion: |  
```

Comment:
Do we (eventually) want to add this seal back to the MassAccess repo for our own forms?

Reply:
Yeah, that wasn't intentionally removed from MassAccess. I'll add that back in now